.husky/pre-commit

```diff
@@ -2,4 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx --no-install lint-staged
-yarn betterer precommit --cache
+yarn betterer precommit 
```

Comment:
this change will clash with the change I submitted minutes ago

Reply:
yep reverting this